docs-src/concepts/what-is-a-schedule.md

```diff
@@ -8,8 +8,14 @@ tags:
   - explanation
 ssdi:
   - Introduced in Temporal Server version 1.17.0
-  - Available in tctl v1.17 and Temporal CLI
-  - Available in Temporal Cloud via tctl and CLI
+  - Available in Temporal CLI (and tctl v1.17)
+  - Available in Temporal Cloud in Public Preview
+  - Available in [Go SDK](/dev-guide/go/features#schedule-a-workflow) version [1.22.0](https://github.com/temporalio/sdk-go/releases/tag/v1.22.0)
+  - Available in [Java SDK](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/schedules/package-summary.html) version [1.20.0](https://github.com/temporalio/sdk-java/releases/tag/v1.20.0)
+  - Available in [Python SDK](/dev-guide/python/features#schedule-a-workflow) version [1.1.0](https://github.com/temporalio/sdk-python/releases/tag/1.1.0)
+  - Available in [TypeScript SDK](https://github.com/temporalio/samples-typescript/tree/main/schedules#schedules) version [1.5.0](https://github.com/temporalio/sdk-typescript/blob/main/CHANGELOG.md#150---2022-12-07)
+  - Available in [.NET SDK](https://dotnet.temporal.io/api/Temporalio.Client.Schedules.html) version [0.1.0](https://github.com/temporalio/sdk-dotnet/releases/tag/0.1.0-alpha4)
+  - Available in [gRPC API](https://api-docs.temporal.io/#temporal.api.workflowservice.v1.CreateScheduleRequest)
```

Comment:
I don't think we should start saying, in these docs, which SDK versions a feature becomes available in. That it's available in the latest stable SDK is good enough for these docs IMO.

These docs are not changelogs. A simple "available as a public preview since server 1.17.0" is good enough I think. The SDK doesn't need to come into play here anymore (a PHP user can come ask us just like they can with other things PHP doesn't support).